client/my-sites/site-settings/section-general.jsx

```diff
@@ -21,7 +21,7 @@ const SiteSettingsGeneral = ( {
 		<div className="site-settings__main general-settings">
 			<GeneralForm site={ site } />
 
-			{ config.isEnabled( 'manage/site-settings/delete-site' ) && site && ! site.jetpack && ! site.is_vip &&
+			{ config.isEnabled( 'manage/site-settings/delete-site' ) && site &&
```

Comment:
Side note: do you think it makes sense to remove the entire `SiteTools` section if `manage/site-settings/delete-site` isn't enabled? I'd expect only the "delete site" link to be disabled, and not the entire Site Tools section.